src/modules/auth/MiniAuth.js

```diff
@@ -26,21 +25,22 @@ const MiniAuth = () => {
   const [state, dispatch] = useContext(AppContext);
 
   useEffect(async () => {
-    if (isAuthenticated) { // TODO we should probably use isAuthenticated here, according to their docs
+    // if (state.currentUser?.type === 'user') return
+    if (isAuthenticated) {
       const userDetails = {
         name: user.name,
```

Comment:
we may need to add some logic in here bc there was an issue with Auth0 not returning a name with signup (with your own email, not via gmail). On refresh it will automatically make the email the name. i.e: `name: user.name ? user.name : user.email.split('@')[0]`